os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/example/ble_peripheral/ble_tizenrt_app.c

```diff
@@ -76,30 +76,6 @@ int ble_tizenrt_app_handle_upstream_msg(uint16_t subtype, void *pdata)
 		case BLE_TIZENRT_MSG_STOP_ADV:
 			ret = le_adv_stop();
 			break;
-        case BLE_TIZENRT_MSG_START_DIRECT_ADV:
```

Comment:
Please add all the clean up changes in a separate commit.

Reply:
This clean up is relative to this new add api.